iphone/cli/commands/_buildModule.js

```diff
@@ -915,12 +915,11 @@ iOSModuleBuilder.prototype.packageModule = function packageModule(next) {
 
 		// 3. platform folder
 		if (fs.existsSync(this.platformDir)) {
-			this.dirWalker(this.platformDir, function (file, name) {
-				var stat = fs.statSync(file);
-				if (name !== 'README.md') {
-					dest.append(fs.createReadStream(file), { name: path.join(moduleFolders, 'platform', path.relative(this.platformDir, file)), mode: stat.mode });
-				}
-			}.bind(this));
+			dest.directory(
+				this.platformDir,
+				path.join(moduleFolders, 'platform'),
+				(entryData) => entryData.name === 'README.md' ? false : entryData
```

Comment:
So, did you test it? Does the `README.md` get excluded from the zip file?

Reply:
yes!
when only the 'README.md' is in the /platform folder (no xcframework as content), the module builds and in the zip the platform folder is also not included.
when  'README.md' and xcframework is included in the platform folder, only the xcframework is in the zipped module included.

works very well!